martor/templates/martor/editor.html

```diff
@@ -17,6 +17,22 @@
     <div class="ui bottom attached tab segment martor-preview" data-tab="preview-tab-{{ field_name }}">
       <p>{% trans "Nothing to preview" %}</p>
     </div>
+    {% if editor_msg or button_text %}
```

Comment:
I think every time we want this behaviour, we should have some explanation for why the button is disabled. Could we make the `editor_msg` mandatory if we use this behaviour? Similarly, I don't see a clear use case for this message if we don't require preview, so maybe make it so either both `button_text` and `editor_msg` are specified or none of them. (would also make the comment below make more sense)

Reply:
done, i changed it so that you need to specify both or neither. i used `ImproperlyConfigured` for the wrong use.